hotel-admin-frontend/src/App.js

```diff
@@ -1,25 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
```

Comment:
import HomePage from './pages' // also make index.js inside pages and export all page from there